pcdsdevices/pim.py

```diff
@@ -372,32 +372,43 @@ class PPMPowerMeter(BaseInterface, Device):
     calibrated for each power meter in units of volts per watt
     is used to calculate the actual energy in the following way:
 
-    `calibrated_mj` = 1000 * (Signal - Background) / (Responsivity * Beam_Rate)
-    """
+    `calibrated_mj` = (Signal - Background) / (Responsivity * Beam_Rate)
+    """
+    def __init__(self, prefix, *, name, **kwargs):
+        import sympy.physics.units as units
+        from sympy.physics.units.prefixes import micro, milli
+        from sympy.physics.units.quantities import Quantity
```

Comment:
We should avoid importing modules inside of class instance methods, unless there's a very specific reason not to.  This is a base class that gets inherited in many later devices, and we'll be re-importing these modules each time one of these classes is instantiated

Reply:
I removed all of this from the class